app/src/main/java/com/aguesoftguar/medalarm/addeditmedicine/AddEditMedicinePresenter.java

```diff
@@ -72,4 +86,43 @@ private void createMedicine() {
    private void updateMedicine() {
       // TODO Handle medicine fields errors or update medicine in Firebase database
    }
+
+   /**
+    * Creates a patient.
```

Comment:
Missing enter after this sentence. It is automatically added if you indent the code using the Android Studio option Code/Reformat code.